mailpoet/tests/integration/AutomaticEmails/WooCommerce/Events/AbandonedCartTest.php

```diff
@@ -162,6 +162,17 @@ public function testItFindsUserByWordPressSession() {
     $this->assertCount(1, $this->scheduledTasksRepository->findAll());
   }
 
+  public function testItSchedulesAbandonedCartAlsoForNonSubscribedSubscribers() {
+    $this->createNewsletter();
+    $this->createSubscriberAsCurrentUser('unconfirmed');
```

Comment:
I would personally use a constant `SubscriberEntity::STATUS_UNCONFIRMED` so that it is easier to find all the places in our codebase where we use the status or eventually update it (very unlikely).

```suggestion
    $this->createSubscriberAsCurrentUser(SubscriberEntity::STATUS_UNCONFIRMED);
```

Reply:
[2c966a7](https://github.com/mailpoet/mailpoet/pull/5669/commits/2c966a7cfcd7daf91da1eefb080bc7545bbcd2bd)